recipes/repository.rb

```diff
@@ -71,14 +79,19 @@
     end
   end
 
+  case node['datadog']['agent_major_version'].to_i
+  when 6, 7
+    baseurl = URI.join(node['datadog']['yumrepo'], "#{node['datadog']['agent_major_version']}/", 'x86_64/').to_s
```

Comment:
we could start using `node['kernel']['machine']` here instead of hardcoding `x86_64` (if that helps with ARM support on RHEL-like platforms, should be checked).

Not necessary for this PR, just wanted to point it out.